internal/data-importer/cmd/pxc/import.go

```diff
@@ -110,7 +112,11 @@ func runPXCImport(ctx context.Context, configPath string) error {
 		}
 	}()
 
-	pxcRestoreName := "data-import-" + dbName
+	// Create an 8-character long hash for the restore name.
+	// This is done to ensure that long DB names are shortened to not
+	// exceed the 63-character limit for Kubernetes labels.
+	nameHash := md5.Sum([]byte("data-import-" + dbName))
```

Comment:
I would suggest to move it to some common function in order to avoid duplication. We don't have a good ready-to-use file to place it into though... we could have something for common funcitons under `internal/data-importer/cmd`. Up to you, not so critical